### PR TITLE
Ignore some strings while calculating string constant overuse

### LINKED ISSUE #1534: Add Exceptions for String Constant Overuse
# Rule request

I feel it would be good to be able to declare exceptions for WPS226

## Thesis / Reasoning

In a file on a current project, I tend to use `" ".join(iterable)` a lot.
It feels unnatural to declare a CONSTANT for that.
Also, I don't think it is good in this case to disable WPS226 for the entire file as there could be other common string constants.

Alternative: Excluse `" "`, `""`, "`\n"` & friends from this by default.

### PR DESCRIPTION
**Pull request by lensvol**
# I have made things!

## Checklist

- [X] I have double-checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [X] I have created at least one test case for the changes I have made
- [X] I have updated the documentation for the changes I have made
- [X] I have added my changes to the `CHANGELOG.md`

## Related issues

Closes #1534 

🙏 Please, if you or your company is finding wemake-python-styleguide valuable, help us sustain the project by sponsoring it transparently on https://opencollective.com/wemake-python-styleguide. As a thank you, your profile/company logo will be added to our main README which receives hundreds of unique visitors per day.

### CODE REVIEW

File: wemake_python_styleguide/visitors/ast/complexity/overuses.py [hunk 1]
```diff
@@ -22,7 +22,25 @@
...
 class StringOveruseVisitor(base.BaseNodeVisitor):
-    """Restricts several string usages."""
+    """
+    Restricts repeated usage of the same string constant.
+
+    NB: Some short strings are ignored, as their use is very common and
+    forcing assignment would not make much sense (i.e. newlines or "").
+    """
+
+    _ignored_string_constants: Set[Union[str, bytes]] = {
```
Comment (sobolevn) [inline wemake_python_styleguide/visitors/ast/complexity/overuses.py:32]: We use `frozenset` for this kind of constants.

**Review by sobolevn — CHANGES_REQUESTED** [review]
Thanks! ✋

File: wemake_python_styleguide/visitors/ast/complexity/overuses.py (hunk 1 above)
Comment (lensvol) [reply wemake_python_styleguide/visitors/ast/complexity/overuses.py:32]: Okay! Will fix that.

**Review by sobolevn — APPROVED** [review]

**Comment by sobolevn** [comment]
Thanks a lot!

**Pull request merged**